MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -4171,3 +4165,13 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 uint64_t mvkGetRegistryID(id<MTLDevice> mtlDevice) {
 	return [mtlDevice respondsToSelector: @selector(registryID)] ? mtlDevice.registryID : 0;
 }
+
+bool mvkSupportsBCTextureCompression(id<MTLDevice> mtlDevice) {
+#if MVK_MACOS && !MVK_MACCAT
+	if ([mtlDevice respondsToSelector: @selector(supportsBCTextureCompression)]) {
+		return mtlDevice.supportsBCTextureCompression;
+	}
+	return true;
+#endif
+	return false;
```

Comment:
```suggestion
#else
	return false;
#endif
```
I'm actually surprised the way you originally had it doesn't warn.

Reply:
Yes. Odd that clang missed it. Good eye! Fixed now.